### PR TITLE
[python] Format python uss qualifier code

### PR DESCRIPTION
**Pull request by barroco**
This PR introduces a python formatter ([black](https://github.com/psf/black)) to the USS qualifier module. (A future PR will be used to cover other modules)

A new target has been added to the [Makefile](https://github.com/swiss-foca/dss/blob/f1fee46c91ec11774c9a83d22a818adf199207bc/monitoring/uss_qualifier/Makefile): `make format`

After merge of this PR, open PRs and existing branches will certainly need to be rebased / merged and will generate conflicts. Applying the formatter to the changes not merged yet to master should hopefully make conflict resolution easy.

### CODE REVIEW

File: monitoring/uss_qualifier/Makefile [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
 .PHONY: test
 test: lint
 	./scripts/test_docker_fully_mocked.sh
 
 .PHONY: lint
 lint:
+	black --check .
 	find . -name '*.sh' | xargs docker run --rm -v $(CURDIR):/code -w /code koalaman/shellcheck
 
+.PHONY: format
```
Comment (BenjaminPelletier) [inline monitoring/uss_qualifier/Makefile:17]: Can we document that developers should use this to fix issues detected by the `lint` target above in the CI?  (assuming that understanding is correct)

File: monitoring/uss_qualifier/Makefile [hunk 2]
```diff
@@ -1,9 +1,19 @@
+.PHONY: ci
+ci: install_dependencies test
+
+.PHONY: install_dependencies
+install_dependencies:
+	pip install black
```
[the remark this answers is not in the document]
Comment (barroco) [reply monitoring/uss_qualifier/Makefile]: I updated the makefile to use the recommended [docker image](https://black.readthedocs.io/en/stable/usage_and_configuration/black_docker_image.html).

File: monitoring/uss_qualifier/Makefile (hunk 1 above)
Comment (barroco) [reply monitoring/uss_qualifier/Makefile:10]: Please let me know if documenting in [DEVELOPMENT.md](https://github.com/interuss/dss/blob/9f3227d750658e7ecf81235421d44e9f47acb652/DEVELOPMENT.md?plain=1#L18-L20) and [printing an error message](https://github.com/interuss/dss/blob/9f3227d750658e7ecf81235421d44e9f47acb652/monitoring/uss_qualifier/Makefile#L7) address your remark ?

Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/Makefile:10]: Yep, that looks great; thanks!

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**